### ISSUE TITLE
Improve colors and animations of Placeholders component

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#790]
We developed a basic Orange design for the [Placeholders component in v5.1.0](https://boosted.orange.com/docs/5.1/components/placeholders/).

Designers would like to improve the rendering:
* Try to make the glow animation to follow those steps: `#eee` -> `#ddd` -> `#ccc` -> `#999`. It could be hard to do because the actual system of placeholders is based on opacity.
* Try to improve the wave animation that is not really visible.

**Comment by ffoodd** [comment]
Grays are only black on white with reduced opacity: you can make a few tries of reduced black and check the resulting hex code using Firefox eye dropper for exemple.

And there's probably some comparison table out there on the internet 🧐

**Comment by julien-deramond** [comment]
Yep that's exactly what I've began to do 😄 but I didn't want to block the release of the v5.1.0 by waiting the result of this study in collaboration with the designers.
It was a bit tedious but I think it could work.

**Comment by MaxLardenois** [comment]
Boosted only
